__tests__/plugin-test.js

```diff
@@ -1,8 +1,8 @@
-const fs = require('fs');
-const { createTempDir } = require('broccoli-test-helper');
-const { factory, runTasks } = require('release-it/test/util');
-const Shell = require('release-it/lib/shell');
-const Plugin = require('../index');
+import fs from 'fs';
+import _ from 'lodash';
+import { createTempDir } from 'broccoli-test-helper';
+import { factory, runTasks } from 'release-it/test/util';
+import Plugin from '../index';
```

Comment:
Relative imports need a file extension.

```suggestion
import Plugin from '../index.js';
```

Reply:
Fixed.